File: app/src/main/java/com/example/learn/ui/appbarlayout/AppbarLayoutPagerAdapter.java
package com.example.learn.ui.appbarlayout;

import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentPagerAdapter;

import java.util.ArrayList;
import java.util.List;

public class AppbarLayoutPagerAdapter extends FragmentPagerAdapter {

    List<Fragment> list = new ArrayList<>();

    public AppbarLayoutPagerAdapter(FragmentManager fm) {
        super(fm);
        list.add(new AppbarLayoutFragment());
        list.add(new AppbarLayoutFragment());
        list.add(new AppbarLayoutFragment());
    }

    @Override
    public Fragment getItem(int position) {
        return list.get(position);
    }

    @Override
    public int getCount() {
        return list.size();
    }
}

File: app/src/main/java/com/example/learn/ui/appbarlayout/AppbarLayoutPagerAdapter.kt
package com.example.learn.ui.appbarlayout

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentPagerAdapter
import java.util.*

class AppbarLayoutPagerAdapter(fm: FragmentManager) : FragmentPagerAdapter(fm) {

    var list = ArrayList<Fragment>()

    init {
        list.add(AppbarLayoutFragment())
        list.add(AppbarLayoutFragment())
        list.add(AppbarLayoutFragment())
    }

    override fun getItem(position: Int): Fragment {
        return list[position]
    }

    override fun getCount(): Int {
        return list.size
    }


}